src/Worker.ts

```diff
@@ -0,0 +1,36 @@
+import { ref, onMounted, onUnmounted, Ref } from '@vue/composition-api';
+
+export function useWorker(url: string) {
+  let message: Ref<any> = ref(null); 
```

Comment:
I get why `message` was confusing, so we could change it to `data` instead so it is clear that's not a function but rather a data reference.

This will make it more aligned with the `useWebSocket` function.

Reply:
Sure. 
The logic is correct, isn't it?

Reply:
Yep, you fixed it :)